### PR TITLE
Touches ended

### PR DESCRIPTION
**Pull request by marcussatellite**
removed unnecessary code from touchesEnded throughout entire app.
whitespace.
higher order swift.

### CODE REVIEW

File: AudioKitSynthOne/Keyboard/UI Components/KeyboardView+Touches.swift [hunk 1]
```diff
@@ -162,19 +146,11 @@ extension KeyboardView {
 
         // check that current touches conforms to onKeys, remove stuck notes
         let notes = notesFromTouches(touches ?? Set<UITouch>() )
-        let disjunct = onKeys.subtracting(notes)
-        if !disjunct.isEmpty {
-            for note in disjunct {
-                pressRemoved(note)
-            }
-        }
+        onKeys.subtracting(notes).forEach { pressRemoved($0) }
```
Comment (Prabhakar-iOS) [inline AudioKitSynthOne/Keyboard/UI Components/KeyboardView+Touches.swift]: 💯

File: AudioKitSynthOne/Sequencer/UI Components/RateKnob.swift [hunk 2]
```diff
@@ -10,12 +10,12 @@ import UIKit
 
 public class RateKnob: MIDIKnob {
 
+    private var _value: Double = 0
```
Comment (Prabhakar-iOS) [inline AudioKitSynthOne/Sequencer/UI Components/RateKnob.swift]: Can we avoid using `_` before variable name?

**Comment by sieren** [comment]
Small observation:
```override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
for _ in touches {
  value = 1 - value 
  setValueCallback(value)
}
```

is there any reason we're having the Knobs react to multiple touches?
In this case I'd say let's get rid of the for loop and do a `guard let touch = touches.first else { return }` followed by the value callback/value calc.

File: AudioKitSynthOne/Sequencer/UI Components/RateKnob.swift (hunk 2 above)
Comment (marcussatellite) [reply AudioKitSynthOne/Sequencer/UI Components/RateKnob.swift]: done

**Pull request merged**